alcs-frontend/src/app/features/board/card-detail-dialog/card-detail-dialog.component.html

```diff
@@ -1,78 +1,115 @@
-<div mat-dialog-title class="header">
-  <div>
-    <a href="application/{{application.fileNumber}}" target="_blank" rel="noreferrer">
-      <h2 class="card-title">{{ application.fileNumber }} ({{ application.applicant }})</h2>
-    </a>
-    <span>
-      in <span class="italic card-status-label">{{ application.statusDetails.label }}</span>
-    </span>
+<div mat-dialog-title>
+  <div class="close">
+    <button mat-icon-button [mat-dialog-close]="isApplicationDirty">
+      <mat-icon>close</mat-icon>
+    </button>
   </div>
-  <div class="center">
-    <div class="card-state">{{ application.paused ? 'Paused' : 'Active' }}</div>
+  <div class="header-row">
+    <div class="left">
+      <h3 class="card-title">{{ application.fileNumber }} ({{ application.applicant }})</h3>
+      <div
+        class="center application-type"
+        [ngStyle]="{
+          backgroundColor: application.typeDetails.backgroundColor,
+          color: application.typeDetails.textColor
+        }"
+      >
+        {{ application.typeDetails.label }}
+      </div>
+    </div>
     <div>
-      <button (click)="onTogglePriority()" mat-icon-button>
-        <span
+      <button color="accent" [matMenuTriggerFor]="moreActions" mat-flat-button>More Actions ▾</button>
+      <mat-menu xPosition="before" #moreActions="matMenu">
+        <button
+          [mat-dialog-close]="isApplicationDirty"
+          [routerLink]="['application', application.fileNumber]"
+          mat-menu-item
+        >
+          View Detail
+        </button>
+        <button
+          [mat-dialog-close]="isApplicationDirty"
+          [routerLink]="['application', application.fileNumber]"
+          mat-menu-item
+        >
+          Add Info Request
+        </button>
+        <button
+          [mat-dialog-close]="isApplicationDirty"
+          [routerLink]="['application', application.fileNumber, 'site-visit-meeting']"
+          mat-menu-item
+        >
+          Add Site Visit / Meeting
+        </button>
+      </mat-menu>
+    </div>
+  </div>
+  <div>
+    <span class="region">{{ application.regionDetails?.label }} Region</span>
```

Comment:
If this needs a safe traversal operator, is there a chance it doesn't exist? What should this display if so?

Reply:
Region is only nullable temporarily, as fallout of doing the new card ui before doing the new card creation UI. Once we complete the new card creation UI, this will no longer be nullable and will always be displayed.

Reply:
Let's remember to change it afterwards then